pymatgen/analysis/tests/test_phase_diagram.py

```diff
@@ -621,6 +623,14 @@ def test_read_json(self):
             dumpfn(self.pd, "pd.json")
             loadfn("pd.json")
 
+    def test_el_refs(self):
+        # load a pre_computed phase diagram,
+        # which was retrieved via new API and serialized to the MSONable object
+        with open(module_dir / "pre_computed_phase_diagram_Li-Fe-O.json") as pd:
+            phaseDiagram = json.load(pd, cls=MontyDecoder)
+        # check the keys in el_refs dict have been updated to Element object
+        assert str not in [type(el) for el in phaseDiagram.el_refs.keys()]
```

Comment:
```suggestion
        assert all(isinstance(el, Element) for el in phaseDiagram.el_refs)
```

Reply:
I noticed the pre_computed_phase_diagram_Li-Fe-O.json is about 25Mb. I deleted it and utilized existed entries to make an imitation of pre_computed phase diagram with the KeyError issue. Thanks!